.vscode/launch.json

```diff
@@ -18,6 +18,15 @@
             "args": [
                 "--useOutClusterClient=false"
             ]
+        },
```

Comment:
Why do you keep adding these lauch configs. Doesnt go tools work with you vscode? You should see a link that will allow you to run a single test or all tests in a package

Reply:
It's the easiest way for me to test. The terminal takes time writing in and copy pasting the full project/package path

Reply:
Removed them in this PR